config/locales/es.yml

```diff
@@ -0,0 +1,15 @@
+es:
+  activerecord:
+    attributes:
+      book:
+        genre: "genero"
+        author: "autor"
+        publisher: "publicador"
+        year: "año"
+        title: "titulo"
+  book: 'libro'
+  user: 'usuario'
+  rent: 'alquiler'
+  from: 'desde'
```

Comment:
aren't this one and another ones attributes of the rent ? I would put it in the same way you put it with the books attributes